tests/codeception/functional/LockupSettings/LockupSettingsCest.php

```diff
@@ -0,0 +1,220 @@
+<?php
+
+/**
+ * Test for the lockup settings
+ */
+class LockupSettingsCest {
```

Comment:
Please expand on the tests to include tests for the logo override and for access to the config page form itself.

1. Test that admin can upload a new logo and have it show up on the home page
2. Test that contributors cannot access the lockup form
3. Test that editors cannot access the lockup form
4. Test that the site_manager can access the lockup form

Reply:
ok